app/script/auth/localeConfig.js

```diff
@@ -10,6 +11,9 @@ function getLocale() {
 
 export function currentLanguage() {
   const LANGUAGE_SHORTHAND_LENGTH = 2;
-  const languageParam = getURLParameter('hl');
-  return languageParam || getLocale().substr(0, LANGUAGE_SHORTHAND_LENGTH) || DEFAULT_LANGUAGE;
+  return getURLParameter('hl') || getLocale().substr(0, LANGUAGE_SHORTHAND_LENGTH) || DEFAULT_LANGUAGE;
+}
+
+export function currentCurrency() {
+  return getURLParameter('currency') || DEFAULT_CURRENCY;
```

Comment:
Time to build an enum with the URL params?

Reply:
Good idea!